managed/ui/src/routes.js

```diff
@@ -121,6 +121,10 @@ function validateSession(store, replacePath, callback) {
   const userId = Cookies.get('userId') || localStorage.getItem('userId');
   const customerId = Cookies.get('customerId') || localStorage.getItem('customerId');
   if (_.isEmpty(customerId) || _.isEmpty(userId)) {
+    var location = window.location.pathname;
+    if (location !== '/') {
+      localStorage.setItem('pathToRedirect', location);
```

Comment:
can we use `initialPath` instead of  `pathToRedirect`?

Reply:
For now, I am leaving this as is, let's hear from others.

Reply:
Cool. Just keep it in mind.

Reply:
actually, that is the last visited page before possible the redirect to the login page, so smth like `lastVisitedPage` better reflects its value

Reply:
One more note - I'd suggest using `sessionStorage` to avoid redirecting to stale URL as localStorage persists value indefinitely

Reply:
Yeah, I will change that, but for the same reason I am clearing it out once it is redirected to the intended page.

Reply:
My suggestion is to put intented url in querystring , for e.g.  http://localhost:3000/login?redirectUrl=<intendedUrl>

Reply:
+1 to @sunilsutarDCE @nishantSharma459 Look at my previous comment, we should use redirectURL or something similar in the query parameter.

Reply:
Sure @sunilsutarDCE @mjoshi0923 I will do the needful.